src/main/java/org/elasticsearch/search/highlight/HighlightPhase.java

```diff
@@ -46,6 +45,7 @@
 public class HighlightPhase extends AbstractComponent implements FetchSubPhase {
 
     private final Highlighters highlighters;
+    private static final ImmutableList<String> sortedHighlighters = ImmutableList.of("fvh", "postings", "plain");
```

Comment:
Yeah - at least needs the ALL_UPPER_CASE naming. Probably should be moved to the top of the class too because that's where I usually look for static stuff.